src/python/WMCore/WMSpec/WMTask.py

```diff
@@ -532,26 +532,22 @@ def setJobResourceInformation(self, timePerEvent=None, sizePerEvent=None, memory
         the three key values are main memory usage, time per processing unit (e.g. time per event) and
         disk usage per processing unit (e.g. size per event).
         """
+        if self.taskType() in ["Merge", "Harvesting", "Cleanup", "LogCollect"]:
```

Comment:
BTW, it looks like we have never set these requirements for Harvesting tasks. Which means, even DQMHarvest workflows would be missing this memory setup.

For taskchain, it would be actually set to None, since there is no taskname (it's not a TaskChain task, though it's a task of the WMWorkload). For DQMHarvest workflow it would work because this setting would be just an integer.
I'm removing this filter "Harvesting" in this line.